<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LobbyWindow</class>
 <widget class="QMainWindow" name="LobbyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedPages">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>401</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="MainMenu">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="Titulo">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Arial Rounded MT Bold</family>
          </font>
         </property>
         <property name="text">
          <string>Need for Speed</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="JugarCotainer">
       <item>
        <widget class="QPushButton" name="playButton">
         <property name="text">
          <string>Jugar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="CerrarConteiner">
       <item>
        <widget class="QPushButton" name="quitButton">
         <property name="text">
          <string>Cerrar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="LobbySelector">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>hola</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
